app/controllers/candidate_interface/personal_details/nationalities_controller.rb

```diff
@@ -69,6 +69,14 @@ def nationalities_params
       def british_or_irish?
         (NationalitiesForm::UK_AND_IRISH_NATIONALITIES & current_application.nationalities).present?
       end
+
+      def right_to_work_path(application_form)
+        if application_form.recruitment_cycle_year >= NationalitiesForm::NEW_RIGHT_TO_WORK_FLOW_STARTS
```

Comment:
have we thought about how we're going to handle this for carried over applications?

Reply:
yes there needs to be some extra logic in `DuplicateApplication` (I think) to reset the completed flag to false.

Reply:
Amazing. We're going to have to think about what we're going to do with people who have already carried over as well since we're past the apply1 deadline. Probs just a backfill.

It's looking really great overall Steve! Great work